build.sbt

```diff
@@ -162,7 +162,8 @@ lazy val query = (project in file("query"))
         oldStrategy(x)
     },
     parallelExecution in Test := false,
-    libraryDependencies ++= configDeps ++ akkaPersistenceDeps ++ slickDeps
+    fork in Test := true,
```

Comment:
Source: I learned more about this error by reading [this GH issue from SBT](https://github.com/sbt/sbt/issues/653).